zerver/data_import/rocketchat.py

```diff
@@ -747,8 +748,11 @@ def message_to_dict(message: Dict[str, Any]) -> Dict[str, Any]:
 
                 zulip_mention = f"#**{parent_stream_name}>{converted_topic_name}**"
 
-            mention_data = {"rc_mention": rc_mention, "zulip_mention": zulip_mention}
-            rc_channel_mention_data.append(mention_data)
+            if zulip_mention == "":
```

Comment:
I think `zulip_mention` can be the empty string only if `mention_rc_channel_id` is in neither map above; I think it might be better to do this special logic as an `else` clause that logs and continues.

Reply:
I've added the `else` clause with a continue since I don't know the implications of `unknown` for the mention attribute.